### PR TITLE
Vet clinic

### PR DESCRIPTION
**Pull request by mhnazary**
In this Project I did:
Create a database named vet_clinic.
Create a table named animals with the following columns:
id: integer
name: string
date_of_birth: date
escape_attempts: integer
neutered: boolean
weight_kg: decimal
Copy the SQL you used in the previous point into the schema.sql file in the generated repository.
Insert the following data:
Animal: His name is Agumon. He was born on Feb 3rd, 2020, and currently weighs 10.23kg. He was neutered and he has never tried to escape.
Animal: Her name is Gabumon. She was born on Nov 15th, 2018, and currently weighs 8kg. She is neutered and she has tried to escape 2 times.
Animal: His name is Pikachu. He was born on Jan 7th, 2021, and currently weighs 15.04kg. He was not neutered and he has tried to escape once.
Animal: Her name is Devimon. She was born on May 12th, 2017, and currently weighs 11kg. She is neutered and she has tried to escape 5 times.
Copy the SQL you used in the previous point into the data.sql file in the generated repository.
Write queries for the following quests:
Find all animals whose name ends in "mon".
image

List the name of all animals born between 2016 and 2019.
image

List the name of all animals that are neutered and have less than 3 escape attempts.
image

List the date of birth of all animals named either "Agumon" or "Pikachu".
image

List name and escape attempts of animals that weigh more than 10.5kg
image

Find all animals that are neutered.
image

Find all animals not named Gabumon.
image

Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
image

Take a screenshot of the results of your queries.
Copy the SQL you used in the previous point into the queries.sql file in the generated repository.
Remember to add the screenshot with the results of your queries to your Pull Request description.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+SELECT FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered=true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+ SELECT * FROM animals WHERE neutered = true;
+ SELECT * FROM animals WHERE name != 'Gabumon';
+ SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```
Comment (usorfaitheloho) [inline queries.sql]: you did well with your database project, kindly note that you didn't write the first query for this :

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @mhnazary  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Created database named `vet_clinic` ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,249 @@
...
+## Getting Started
+
+This repository includes files with plain SQL that can be used to recreate a database:
+
+- Use [schema.sql](./schema.sql) to create all tables.
+- Use [data.sql](./data.sql) to populate tables with sample data.
+- Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
+
+<a name="readme-top"></a>
+
```
Comment (rloterh) [inline README.md:12]: - [ ] [OPTIONAL]
Let's endeavor to remove or update the underlined aspects of the ReadMe to make the ReadMe appear more professional!

image

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Mahram,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Good job on intruding improvements to the query!

## Optional suggestions 🧰 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**